File: Competitor Answers/Feyyaz Talha ÜLGER/soru-8.py
k1 = int(input("İlk kenarın uzunluğunu giriniz: "))
k2 = int(input("İkinci kenarın uzunluğunu giriniz: "))
k3 = int(input("Üçüncü kenarın uzunluğunu giriniz: "))
k4 = int(input("Dördüncü kenarın uzunluğunu giriniz: "))
h = k1


if k1 == k2 and k1 == k3 and k1 == k4 and k2 == k3 and k2 == k4 and k3 == k4:
    print("Karedir")
else:
    print("Dikdörtgendir")

File: Competitor Answers/Hüseyin Bora İÇÖZ - Ramazan Semih UĞUR/soru8.cpp
#include <stdio.h>
#include <stdlib.h>

void KareDikdortgen(int a,int b,int c,int d){
	if(a==b!=0&&a==c!=0&&a==d!=0)
	{
		printf("Karedir");
	}
  
    else if((a==b!=0&&c==d!=0&&a!=c)||(a==c!=0&&b==d!=0&&a!=b)||a==d!=0&&b==c!=0&&a!=b)
    {
    	printf("Dikdörtgendir");	
	}
	
	else
	printf("Hatali Giris!");
}

int main()
{
printf("Kenar Uzunluklarýný Giriniz:");
int a,b,c,d;
scanf("%d %d %d %d",&a,&b,&c,&d);	
KareDikdortgen(a,b,c,d);	
}
